android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1839,7 +1879,65 @@ public void setPausedModifier(boolean paused) {
             } else {
                 pausePlayback();
             }
+            updatePictureInPictureActions(paused);
+        }
+    }
+
+    public void setPictureInPicture(boolean pictureInPictureEnabled) {
+        this.pictureInPictureEnabled = Build.VERSION.SDK_INT >= Build.VERSION_CODES.N && pictureInPictureEnabled;
+    }
```

Comment:
With `react-native-video@6.0.0-beta.5` on iOS, when we change `pictureInPicture` props from `false` to `true` video enter pip mode.
This allow me (us?) to have a custom pipButton that manually enter in pip mode.

With this implementation on android, we enter on pip mode only if we set `pictureInPicture` to true and exit application

I think we can have same behavior as iOS with these code, let me know what you think

```suggestion
    public void setPictureInPicture(boolean pictureInPictureEnabled) {
        boolean newPictureInPictureEnabled = Build.VERSION.SDK_INT >= Build.VERSION_CODES.N && pictureInPictureEnabled;
        boolean oldPictureInPictureEnabled = this.pictureInPictureEnabled;
        
        this.pictureInPictureEnabled = newPictureInPictureEnabled;

        if (!oldPictureInPictureEnabled && newPictureInPictureEnabled) {
            enterPictureInPictureMode();
        }
    }
```

Reply:
@gkueny 
Thanks for your review! 👍

I think `pictureInPicture` prop is actually more accurately named `pictureInPictureOnLeave`. However, I wrote the same prop name to be in sync with iOS.

If you need that implementation, I think something like `videoRef.enterPictureInPicture()` or `videoRef.exitPictureInPicture()` would be more appropriate.

Controlling everything with a prop makes complexity.

Reply:
> If you need that implementation, I think something like videoRef.enterPictureInPicture() or videoRef.exitPictureInPicture() would be more appropriate.

Yes I agree
As behavior is not the same as iOS, is it something worth to add in documentation ?

I don't know the opinion of @freeboub & @KrzysztofMoch , but it may be worth it to use `enterPictureInPicture() / exitPictureInPicture()` also for iOS ? & rename `pictureInPicture` props to `pictureInPictureOnLeave` 

I will check to implement `enterPictureInPicture` & `exitPictureInPicture` & will propose an MR if I can make it work

Reply:
Interested in the iOS changes since we'd love to have PiP on both OSs.